### PR TITLE
Gpt model

### PR DESCRIPTION
**Pull request by hexisyztem**

### CODE REVIEW

File: lightseq/csrc/kernels/cuda/gptKernels.cc.cu [hunk 1]
```diff
@@ -33,91 +33,138 @@ real_seq_len: record seq len exclude padding, [batch_size]
...
+// template void ker_gpt_embedding_launcher<float>(
+//     int batch_size, int batch_seq_len, int hidden_size, cudaStream_t stream,
+//     const float* token_emb, const float* pos_emb, const int* token_id,
+//     float* output, int* real_seq_len, int padding_id, int pos_offset);
+
+// template void ker_gpt_embedding_launcher<__half>(
+//     int batch_size, int batch_seq_len, int hidden_size, cudaStream_t stream,
+//     const __half* token_emb, const __half* pos_emb, const int* token_id,
+//     __half* output, int* real_seq_len, int padding_id, int pos_offset);
+
```
Comment (Taka152) [inline lightseq/csrc/kernels/cuda/gptKernels.cc.cu:121]: could remove comment

File: lightseq/csrc/kernels/cuda/gptKernels.cc.cu [hunk 2]
```diff
@@ -33,91 +33,138 @@ real_seq_len: record seq len exclude padding, [batch_size]
...
 template <typename T>
```
Comment (Taka152) [inline lightseq/csrc/kernels/cuda/gptKernels.cc.cu:122]: need to add half version

File: lightseq/csrc/models/includes/transformer.h [hunk 3]
```diff
@@ -11,7 +11,7 @@
 #include "linear_layer.h"
 #include "sample_layer.h"
 #include "encdec_kv_layer.h"
-#include "model_util.h"
+// #include "model_util.h"
```
Comment (Taka152) [inline lightseq/csrc/models/includes/transformer.h:14]: remove comment

File: lightseq/csrc/pybind/pybind_model.cpp [hunk 4]
```diff
@@ -7,19 +7,20 @@
 #include "util.h"
 
 #include "bert.h"
-#include "bert_crf.h"
+// #include "bert_crf.h"
```
Comment (Taka152) [inline lightseq/csrc/pybind/pybind_model.cpp:10]: why delete this model?

Comment (hexisyztem) [reply lightseq/csrc/pybind/pybind_model.cpp:10]: Need to be modified to adapt to the interface code.

File: lightseq/csrc/models/includes/transformer.h (hunk 3 above)
Comment (hexisyztem) [reply lightseq/csrc/models/includes/transformer.h]: resolved

File: lightseq/csrc/kernels/cuda/gptKernels.cc.cu (hunk 2 above)
Comment (hexisyztem) [reply lightseq/csrc/kernels/cuda/gptKernels.cc.cu:122]: Will be fixed in next commit.

File: lightseq/csrc/kernels/cuda/gptKernels.cc.cu (hunk 1 above)
Comment (hexisyztem) [reply lightseq/csrc/kernels/cuda/gptKernels.cc.cu:121]: temporarily reserved

**Review by Taka152 — APPROVED** [review]

**Pull request merged**